apps/taxes/src/lib/graphql.ts

```diff
@@ -43,3 +44,13 @@ export const createClient = (url: string, getAuth: AuthConfig<IAuthState>["getAu
       multipartFetchExchange,
     ],
   });
+
+export const createUrqlClientFromCtx = ({
```

Comment:
please avoid "context" without context :) 

`createClientFromAuthData` will be good

Reply:
done